# snapshot of planning context
Header header
# input
State state
MotSet mot_set
LaneSet lane_set
Freespace freespace